.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--
+
+Thanks for opening an issue! To help the PyGDF team handle your information
+efficiently, please first ensure that there is no other issue present that
+already describes the issue you have
+(search at https://github.com/gpuopenanalytics/pygdf/issues?&q=is%3Aissue).
+
+If there is no issue present please jump to a section below and delete the
+irrelevant one depending on whether you are:
+
+ * Making a feature request.
+ * Reporting a bug.
+
+For more general "how do I do X?" type questions, please speak visit
+http://gpuopenanalytics.com/#/COMMUNITY for links to Slack and Google
+Groups.
+
+-->
+
+## Feature request
+
+<!--
+
+Please include details of the feature you would like to see, why you would
+like to see it/the use case
+
+-->
+
+## Reporting a bug
+
+<!--
+
+Before submitting a bug report please ensure that you can check off these boxes:
+
+-->
+
+- [ ] I am using the latest version of PyGDF from conda or built from `master`.
```

Comment:
I'd really like to capture dependency information as well if possible. Things like CUDA, Pandas, Numpy, Arrow versions that have the potential to break things.

Reply:
Agreed that would be very helpful. I added in two boxes to fill in environment and version information, let me know if you think I have it covered now.